zp-relayer/workers/sentTxWorker.ts

```diff
@@ -90,8 +88,8 @@ export async function createSentTxWorker<T extends EstimationType>(gasPrice: Gas
 
     if (shouldReprocess) {
       logger.info('%s sending this job for re-processing...', logPrefix)
-      await poolTxQueue.add('reprocess', [job.data.txPayload])
-      return [SentTxState.SKIPPED, lastHash, []] as SentTxResult
+      // Error should be caught by `withLoop` to re-run job
+      throw new Error('Transaction was not mined, but nonce increased')
```

Comment:
Consider to rephrase the message: "Ambiguity detected: nonce increased but no respond that transaction was mined"